### DISCUSSION TITLE
Docker image expects to be run as root due to chown instructions

### DISCUSSION THREAD

**Discussion by spuxx-dev** [firefly-iii/firefly-iii#8894] [General]
Hey there! I'm attempting to set up a firefly-iii service right now. I've noticed that the container attempts to run a series of `chown` and `chmod` instructions on startup, which fails because I run the container as a non-root user.

In general, it's considered bad practice and a security risk to run containers as root. Is there any way to make the container not rely on root permissions? The `chown` shoudn't be required in the first place with the volume and permissions on the host filesystem being set up properly, no?

**Comment by spuxx-dev** [comment]
On a side note, I noticed that the helm charts will also run the service as root, which I don't think should be pushed to users as a default: https://github.com/firefly-iii/kubernetes/blob/main/charts/firefly-iii/values.yaml#L91

**Comment by JC5** [comment]
Your suggestions are more than welcome!
